### DISCUSSION TITLE
Auto Assign Reports

### DISCUSSION THREAD

**Discussion by newtarkprojects** [gameserverapp/Platform#175] [Suggestions]
Hey,

Please could there be a way for reports to be auto-assigned to a permission group and notify anyone in that permission group?

For example, if a player has an issue in-game and submits an admin support ticket the ticket would auto-assign itself to the relevant permission group that can handle the report. This would then send a notification to anyone in that group by email or discord.

Thank you,
Gem

**Comment by mvaessen** [comment]
Hi Gem,

Thanks for your suggestion!

I like the idea, both for this particular section (reports) but also for the system at large. I hadn't really considered using the permission groups as a reference for admins, for other parts of the system. Makes me reconsider whether the name `permissions groups` is the right name. Probably `Admin Teams` works better.

We should definitely include your suggestion in Permissions V2.1!
